tests/StacktraceTest.php

```diff
@@ -297,6 +297,25 @@ public function testGetFrameArgumentsDoesNotModifyCapturedArgs(): void
         $this->assertEquals($result['param2']['key'], 'xxxxx');
     }
 
+    public function testPreserveXdebugFrameArgumentNames(): void
```

Comment:
Would it be possible to refactor this test to be similar to the others that use a JSON fixture and the `assertFrameEquals` method?

Reply:
assertFrameEquals does not check frame arguments :/